### PR TITLE
Add output example

### PR DESCRIPTION
**Pull request by Smiranin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,20 +40,33 @@ it's just attribute directive for textarea:
...
+### Output changed content
+    
+    import { Component } from '@angular/core';
+
+    @Component({
+        selector: 'app',
+        template: `
+                <ng2wig
+                    [content]="text1" 
+                    (contentChange)="updateResult($event)">
```
Comment (stevermeister) [inline README.md]: may be to have it even more simple `result = content`

Comment (Smiranin) [inline README.md]: In this way? 
```
<ng2wig [content]="data"
        (contentChange)="result = $event"></ng2wig>
<div [innerHTML]="result"></div>
```

Comment (stevermeister) [inline README.md]: yes

Comment (Smiranin) [inline README.md]: Changed

**Comment by stevermeister** [comment]
merged it separetely

**Pull request closed without merging**